### PR TITLE
chore: improved mock JWT signer and disabled ES256 signature validation

### PR DESCRIPTION
**Pull request by birtony**
Signed-off-by: Anton Biriukov <anton.biriukov@avast.com>

### CODE REVIEW

**Review by DRK3 — APPROVED** [review]

**Comment by rolsonquadras** [comment]
> fix: modified implementation for integration

The commit title is very generic. Can we update it to something meaningful message?

File: cmd/wallet-js-sdk/src/oidc/presentation/openid4vp.js [hunk 1]
```diff
@@ -87,23 +90,29 @@ export class OpenID4VP {
...
-    } catch (e) {
-      console.error(e);
-      throw new Error(
-        "Error initiating OIDC presentation: signature verification failed",
-        e
-      );
-    }
+    // TODO: https://github.com/trustbloc/agent-sdk/issues/449
+    // try {
+    // await jose.compactVerify(encodedRequestToken, publicKey);
```
Comment (sudeshrshetty) [inline cmd/wallet-js-sdk/src/oidc/presentation/openid4vp.js:105]: why it is commented?

File: cmd/wallet-js-sdk/src/oidc/presentation/openid4vp.js [hunk 2]
```diff
@@ -175,36 +179,24 @@ export class OpenID4VP {
...
-    });
-    Object.defineProperty(header, "kid", {
-      value: kid,
-    });
-    Object.defineProperty(header, "typ", {
-      value: "JWT",
-    });
+    header["alg"] = alg;
+    header["kid"] = kid;
+    header["typ"] = "JWT";
```
Comment (sudeshrshetty) [inline cmd/wallet-js-sdk/src/oidc/presentation/openid4vp.js:184]: :)

Comment (sudeshrshetty) [reply cmd/wallet-js-sdk/src/oidc/presentation/openid4vp.js:184]: `const header = new Object();`

can also be written as 

`const header = {}`

Comment (birtony) [reply cmd/wallet-js-sdk/src/oidc/presentation/openid4vp.js:184]: Yep, the `defineProperty` syntax did not allow me to stringify the objects

**Review by rolsonquadras — APPROVED** [review]

**Review by sudeshrshetty — APPROVED** [review]

**Pull request merged**